### DISCUSSION TITLE
Unable to connect to Github Codespace using Jetbrains Gateway

### DISCUSSION THREAD

**Discussion by faiz-spekit** [community/community#70033] [Codespaces]
### Select Topic Area

Bug

### Body

I have tried multiple time to connect to codespace using jetbrains Gateway but it never connects it always gives me an error message saying "A problem occurred attempting to connect to Codespace."

 
<img width="807" alt="image" src="https://github.com/community/community/assets/113900965/a29df590-4920-4823-a38a-a6244c784e1e">

**Comment by mazzarito** [comment]
Use the SSH connection method i've outlined here: https://github.com/orgs/community/discussions/68150
